### PR TITLE
CF-447 Changes to allow content other than cadf:event type

### PR DESCRIPTION
**Pull request by ChandraAddala**
Current product events allow content of type text. This PR is to make changes so that feeds_access also allows them.

### CODE REVIEW

**Comment by ChandraAddala** [comment]
@shintasmith @usnavi Pls review.

File: wadl/cadf.wadl [hunk 1]
```diff
@@ -221,11 +221,16 @@
...
+                           rax:message="Product/Usage events are not allowed in this feed."
+                           path="not(/atom:entry/atom:content/event:event)"/>
                     <param name="uae"
                            style="plain"
                            required="true"
-                           path="/atom:entry/atom:content/cadf:event/cadf:attachments/cadf:attachment/cadf:content/ua:auditData and /atom:entry/atom:content/cadf:event/cadf:attachments/cadf:attachment/cadf:content/ua:auditData[@version = '1']"
-                           rax:message="This feed only allows CADF events. The version and namespace of the CADF attachment content should correspond to user access events." />
+                           path="if (/atom:entry/atom:content/cadf:event) then (/atom:entry/atom:content/cadf:event/cadf:attachments/cadf:attachment/cadf:content/ua:auditData and /atom:entry/atom:content/cadf:event/cadf:attachments/cadf:attachment/cadf:content/ua:auditData[@version = '1']) else true()"
+                           rax:message="The version and namespace of the CADF attachment content should correspond to user access events." />
```
Comment (shintasmith) [inline wadl/cadf.wadl]: This is fine for now. But do you think its worth adding a unit test case to test this feature, where we allow random content to be posted to /*_access/events feeds?

**Comment by shintasmith** [comment]
looks good to me

File: wadl/cadf.wadl (hunk 1 above)
Comment (usnavi) [inline wadl/cadf.wadl]: :+1:  - protects us from having surprises with system regression tests.

**Comment by usnavi** [comment]
lgtm - @ChandraAddala mege this when the build turns green.

File: wadl/cadf.wadl (hunk 1 above)
Comment (ChandraAddala) [inline wadl/cadf.wadl]: Added files for testing random content.

**Pull request merged**